.chloggen/attributes-processor-resource.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: attributesprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adds ability for attributes processor to process resource attribute and data point attribute matching
```

Comment:
This statement is confusing to read, is there a way to simplify this?

Reply:
tried to improve this, is it still confusing?